src/hooks/useChats.ts

```diff
@@ -0,0 +1,23 @@
+import { useFirestoreConnect } from "react-redux-firebase";
+
+import { venueChatsSelector } from "utils/selectors";
+
+import { useVenueId } from "hooks/useVenueId";
+import { useSelector } from "hooks/useSelector";
+
+export const useVenueChatConnect = () => {
+  const venueId = useVenueId();
+
+  useFirestoreConnect({
+    collection: "venues",
+    doc: venueId,
+    subcollections: [{ collection: "chats" }],
+    storeAs: "venueChats",
+  });
+};
+
+export const useVenueChats = () => {
+  useVenueChatConnect();
```

Comment:
[musing] I don't think there is any issue with us calling this multiple times within the app, as I believe the lib will realise it's the same thing, and not 're-subscribe' to it. I was pondering at one stage that we could build our own logic into our 'connect' hooks (eg. `useVenueChatConnect`) that somehow detected if it had already been subscribed, and if so, didn't try again. But if the library does that for us, even better!